base_tier_validation_waiting/models/tier_review.py

```diff
@@ -2,14 +2,25 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
 from odoo import api, fields, models
+from odoo.tools import config
 
 
 class TierReview(models.Model):
     _inherit = "tier.review"
 
+    def _default_status(self):
+        if (
+            not config["test_enable"]
```

Comment:
do we really need this specific check for tests?

Reply:
Yes, without this commit, it would mean that we need to patch a lot of depending modules, to fix their tests.

Another solution I've evaluated is using seperate tests. That might solve the problem with the CI tests here on github.
But it would simply transfer the problem to the persons running an custom CI like odoo.sh.

The method here is not really clean. But it's the least dirty solution.
(Discussed this test issue with a couple of oca contributors at the OXP)
Maybe you have another insight to solve this problem?